src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -33,15 +35,26 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <PrepareForBuildDependsOn>
             $(PrepareForBuildDependsOn);CppWinRTVerifyKitVersion
         </PrepareForBuildDependsOn>
+        <!-- Note: Before* targets run before Compute* targets. -->
```

Comment:
with these changes in triggers, have you checked that all incremental rebuild scenarios still work?  (i.e., delete the xamlmetadataprovider.idl/cpp, touch single idl/cpp files in the project, etc)

Reply:
I checked that incremental build didn't rebuild unnecessary files. I will also verify the scenario's you listed.

Reply:
Delete xamlmetadataprovider.idl as well as touching files work correctly for the test solution.